### PR TITLE
Added provisional remediation to audit rules.

### PR DESCRIPTION
**Pull request by matejak**
The remediation copies part of the recommended audit config that should assure OSPP compliance.

Disadvantage: The remediation fixes more than one rule at time.

Advantage: This is the way how to fix the issue from the audit project's point of view + we can guarantee related audit rule ordering.

### CODE REVIEW

**Comment by yuumasato** [comment]
@matejak I know there aren't merge conflicts, but when I scan and remediate, some rules that should be passing are failing.
To clear out the doubt whether the failures are due to outdated master or current issues, can you please rebase on top of latest master?

File: shared/bash_remediation_functions/create_audit_remediation_file.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+function create_audit_remediation_file {
```
Comment (yuumasato) [inline shared/bash_remediation_functions/create_audit_remediation_file.sh]: The function name should be more specific about what audit rules are being fixed.
Suggestion: `audit_remediation_unsuccessful_file_modification_detailed`

**Comment by matusmarhefka** [comment]
@yuumasato Yes, the branch is not updated and is missing some important commits from the master.

File: shared/bash_remediation_functions/create_audit_remediation_file.sh (hunk 1 above)
Comment (yuumasato) [reply shared/bash_remediation_functions/create_audit_remediation_file.sh]: Also cite source or reference for these audit rules.

**Comment by yuumasato** [comment]
For some reason, the report of the remediation run fails on rules:
- `audit_rules_unsuccessful_file_modification_open`
- `audit_rules_unsuccessful_file_modification_openat`
- `audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order`

But on second scan, these rules pass.

**Comment by matusmarhefka** [comment]
We need to add remediations also for:
* `audit_rules_unsuccessful_file_modification_open_rule_order`
* `audit_rules_unsuccessful_file_modification_openat_rule_order`
* `audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order`

**Comment by matejak** [comment]
Rebased, changed according to input.

**Comment by matusmarhefka** [comment]
All the audit rules are passing after the latest updates :slightly_smiling_face:

**Comment by matusmarhefka** [comment]
@matejak Good job, thank you.

**Comment by yuumasato** [comment]
Also ACK, I understand the solution is not ideal, and these remediations should be improved in the future.

**Pull request merged**